components/konflux-rbac/policies/validate-rolebindings/kustomization.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- invalid-subjects.yaml
```

minor for consistency

```suggestion
- invalid_subjects.yaml
```